.coafile

```diff
@@ -31,6 +31,12 @@ max_line_length = 120
 [all.links]
 bears = InvalidLinkBear
 
+[html]
+bears = HTMLLintBear
+files = **.html
+htmllint_ignore = http_equiv, entities, optional_tag, concerns_separation, names, protocol
```

Comment:
Why are so many rules being ignored?

`http_equiv` seems fixable by replacing https://github.com/coala/community/blob/master/templates/activity.html#L13 by `<meta charset="utf-8">`

`entities` only triggers on https://github.com/coala/community/blob/master/templates/index.html#L11 which to me seems like a bug? I'm not sure why there is a carriage return there, maybe quotes were intended originally?

`protocol` is also fixable across the files.

I haven't manually checked others, but try to not ignore the rules.